addons/mail/static/src/components/message_action_list/message_action_list.xml

```diff
@@ -3,9 +3,9 @@
     <t t-name="mail.MessageActionList" owl="1">
         <div class="o_MessageActionList d-flex" t-on-click="messageActionList and messageActionList.onClick" t-ref="root">
             <t t-if="messageActionList">
-                <Popover t-if="messageActionList.message.hasReactionIcon" class="o_MessageActionList_action o_MessageActionList_actionReaction p-2 fa fa-lg fa-smile-o" position="'top'" titleAttribute="ADD_A_REACTION" t-on-o-emoji-selection="messageActionList.onEmojiSelection" t-on-o-popover-closed="messageActionList.onReactionPopoverClosed" t-on-o-popover-opened="messageActionList.onReactionPopoverOpened" t-ref="reactionPopover">
+                <Popover t-if="messageActionList.message.hasReactionIcon" class="o_MessageActionList_action o_MessageActionList_actionReaction p-2 fa fa-lg fa-smile-o" position="'top'" titleAttribute="ADD_A_REACTION" onClosed="messageActionList.onReactionPopoverClosed" onOpened="messageActionList.onReactionPopoverOpened" t-ref="root">
```

Comment:
t-ref root changed in the wrong PR?
Need to adapt existing usage of `reactionPopover`.

Reply:
wrong rebase, had to be on root node. Fixing it right now